pyresample/_spatial_mp.py

```diff
@@ -238,12 +240,15 @@ def _parallel_proj(scheduler, data1, data2, res1, res2, proj_args, proj_kwargs,
         _res2 = shmem_as_ndarray(res2)
 
         # Initialise pyproj
-        proj = pyproj.Proj(*proj_args, **proj_kwargs)
+        proj_def = proj_args[0] if proj_args else proj_kwargs
+        crs = CRS.from_user_input(proj_def)
+        gcrs = get_geodetic_crs_with_no_datum_shift(crs)
+        transformer = pyproj.Transformer.from_crs(gcrs, crs, always_xy=True)
+        trans_kwargs = {"radians": radians, "errcheck": errcheck, "direction": "INVERSE" if inverse else "FORWARD"}
```

Comment:
A little late, but I wanted to note that using the enumeration for transform direction provides a slight performance improvement.

Reply:
Thanks. Are we talking a line of code difference (to look up the value in the Enum) are like a completely different code path?

Reply:
If you provide the enumeration you skip the overhead of building the enum and the type checks in cython appear to be faster. Related https://github.com/pyproj4/pyproj/pull/1208

Reply:
Could that enum please be added to `pyproj/__init__.py` for easier access? All the other objects are convenience-imported there.

Reply:
Understood. Sounds good.

Reply:
Ah that PEP for lazy importing was rejected. Just noticed.